<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>715</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>660</width>
      <height>660</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>560</width>
      <height>570</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>99999</width>
      <height>99999</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
   </widget>
   <widget class="QListWidget" name="LogList">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="INFOLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>INFO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LOGLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>469</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOG:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="linesLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>lines:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="vexsLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>vexs:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="vexsNum">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="LinesNum">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="DelButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除结点</string>
    </property>
   </widget>
   <widget class="QToolButton" name="setClearButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清空视图</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DijkstraButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dijkstra</string>
    </property>
   </widget>
   <widget class="QToolButton" name="FloydButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>430</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Floyd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>最短路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addBus">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>280</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>添加线路</string>
    </property>
   </widget>
   <widget class="QListWidget" name="busList">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>211</y>
      <width>31</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>经停：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="busStopEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>编号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="busNoEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_changeBus">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>修改线路</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delBus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>删除线路</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clearBus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清空线路</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
